src/geolocation/mod.rs

```diff
@@ -0,0 +1,11 @@
+use solana_client::rpc_response::RpcContactInfo;
+use std::net::IpAddr;
+
+pub mod api;
+pub mod caching;
+
+pub const GEO_DB_CACHE_LOCATION: &str = "geolocation_cache.db";
```

Comment:
Right now, this file path means that after `cargo install`, the cache database will still be saved at where the crate was built (i.e. repo directory). Maybe it should be saved somewhere like `$HOME/.solana-exporter/geolocation_cache.db`?

Reply:
That would be better indeed.

Reply:
Fixed in 93054af